docs/en/reference/supporting-other-databases.rst

```diff
@@ -28,9 +28,9 @@ contributed back to Doctrine to make it an even better product.
 Implementation Steps in Detail
 ------------------------------
 
-1. Add your driver shortcut to class-name `Doctrine\DBAL\DriverManager`.
+1. Add your driver shortcut to class-name ``Doctrine\DBAL\DriverManager``.
```

Comment:
Should it be “to the `Doctrine\DBAL\DriverManager` class”? How is the class _name_ relevant here?

Reply:
Fixed.